### PR TITLE
Fix using invalid attr_count

### PR DESCRIPTION
**Pull request by tvar**
reconstructTupleValues failed with assertion failure, attr_count - count for columns in source COPY command. Here we need attrcount from table, to construct tuple

### CODE REVIEW

**Comment by hlinnaka** [comment]
Got a test case to reproduce this one?

**Comment by tvar** [comment]
enable asserts and copy less, than table column count

File: src/backend/commands/copy.c [hunk 1]
```diff
@@ -4289,9 +4289,7 @@ CopyFrom(CopyState cstate)
 	int			attnum;
 	int			i;
 	Oid			in_func_oid;
-	Datum		*partValues = NULL;
-	bool		*partNulls = NULL;
-	bool		isnull;
+    bool		isnull;
```
Comment (hsyuan) [inline src/backend/commands/copy.c]: the indent for every changed lines should be adjusted to match postgres style.

**Comment by hlinnaka** [comment]
> enable asserts and copy CSV with less, than table column count. Table must be partioned.
> really need script?

Yes please. I tried to reproduce this with:
```
-- Create file with one column.
COPY (SELECT * FROM generate_series(1, 75)) TO '/tmp/copyparttest.csv';

CREATE TABLE copyparttest (id int4, extracol text) PARTITION BY RANGE (id) (START (1) END (100) EVERY (50));

-- Try to load it to a partitioned table with two columns.
COPY copyparttest FROM '/tmp/copyparttest.csv';
COPY copyparttest (id) FROM '/tmp/copyparttest.csv';
```
But it worked fine. I must've misunderstood the case when the assertion failure happens?

**Comment by xinzweb** [comment]
@tvar any update on this issue? This is found by pullreminder from GPDB slack channel https://greenplum.slack.com/messages/C9NHTUWUB. Please close it if this issue is not reproable. Thanks.

**Comment by xinzweb** [comment]
@tvar I haven't heard from you yet. @hlinnaka any additional input on this PR? Since, we don't have a repro, what's the best way to close it? @tvar if you got additional info, please provide it here. @vraghavan78 is this something we can work on with the QP team? Thanks.

**Comment by xinzweb** [comment]
I am closing this PR for now. If there is useful information to follow-up, feel free to re-open it. Thanks.

**Pull request closed without merging**